style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: green;
+}
```

Comment:
- [ ] Kindly fix linter errors currently they are coming from your stylelint configuration, figure out where they are coming from and fix them

Reply:
Hi Lydia,
According to the post linked below, if I understood correctly, the same problem was solved before by deleting 'package-lock.json' or by editing the configuration file linters.yml.
https://questions.microverse.org/t/stylelint-error-cannot-find-module-postcss-lib-input/1360/3
Making changes in linters.yml was strongly not raccomended and I am not sure if I can just delete the package-lock.json file. What would you recommend?